src/sql/workbench/browser/designer/media/designer.css

```diff
@@ -80,7 +80,7 @@
 }
 
 .designer-component .designer-tab-view {
-	overflow: scroll;
+	overflow: auto;
```

Comment:
this has nothing to do with the issue, just something I noticed while debugging the issue. if we use `scroll` the space will always be reserved for the scrollbars, which was never my intention.